{{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/settings.py

```diff
@@ -101,6 +101,7 @@
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
     "django.contrib.sites.middleware.CurrentSiteMiddleware",
+    "{{ cookiecutter.project_name }}.middleware.CachelessAPIMiddleware",
```

Comment:
Should this be `main.middleware. CachelessAPIMiddleware` now?

Reply:
Yes, good catch